src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -132,20 +130,22 @@ export const ConversationSpace: React.FC<ConversationSpaceProps> = ({
               )}
             </div>
           </div>
-          <div className="seated-area">
-            <TablesUserList
-              setSeatedAtTable={setSeatedAtTable}
-              seatedAtTable={seatedAtTable}
-              venueId={venue.id}
-              TableComponent={TableComponent}
-              joinMessage={venue.hideVideo === false}
-              customTables={tables}
-              defaultTables={CONVERSATION_TABLES}
-              showOnlyAvailableTables={showOnlyAvailableTables}
-              venue={venue}
-              template={VenueTemplate.conversationspace}
-            />
-          </div>
+          {/*
+            TODO
+          <TableGrid
```

Comment:
`DISABLED_DUE_TO_` in `disableSettings` is the advised way of "commenting out" code so that it doesn't go stale and out of sync with other changes